[datadogfeature] adding orchestrator explorer as a feature (#178)
DataDog/datadog-operator: docs/configuration.md

| `agent.priorityClassName`                                                                                    | If specified, indicates the pod's priority. "system-node-critical" and "system-cluster-critical" are two special keywords which indicate the highest priorities with the former being the highest priority. Any other name must be defined by creating a PriorityClass object with that name. If not specified, the pod priority will be default or zero if there is no default.                                                                                                                                                                                                                                                                       |

> | `agent.process.enabled`                                                                                      | Enable this to activate the process agent and live container monitoring. Note: /etc/passwd is automatically mounted to allow username resolution. ref: https://docs.datadoghq.com/graphing/infrastructure/process/#kubernetes-daemonset                                                                                                                                                                                                                                                                                                                                                                                                                |

Reviewer: ```suggestion
| `agent.process.enabled`                                                                                      | Enable this to activate the process Agent and live container monitoring. Note: /etc/passwd is automatically mounted to allow username resolution. ref: https://docs.datadoghq.com/graphing/infrastructure/process/#kubernetes-daemonset                                                                                                                                                                                                                                                                                                                                                                                                                |
```

> | `agent.process.processCollection`                                                                            | Enable this to activate the process collection additionally to container collection.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |

Reviewer: ```suggestion
| `agent.process.processCollection`                                                                            | Enable this to activate the process collection in addition to container collection.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

| `clusterAgent.config.clusterChecksEnabled`                                                                   | Enable the Cluster Checks and Endpoint Checks feature on both the cluster-agents and the daemonset ref: https://docs.datadoghq.com/agent/cluster_agent/clusterchecks/ https://docs.datadoghq.com/agent/cluster_agent/endpointschecks/ Autodiscovery via Kube Service annotations is automatically enabled                                                                                                                                                                                                                                                                                                                                              |

> | `clusterAgent.config.collectEvents`                                                                          | Enable this to start event collection from the kubernetes API ref: https://docs.datadoghq.com/agent/cluster_agent/event_collection/                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |

Reviewer: ```suggestion
| `clusterAgent.config.collectEvents`                                                                          | Enable this to start event collection from the Kubernetes API ref: https://docs.datadoghq.com/agent/cluster_agent/event_collection/                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

| `credentials.useSecretBackend`                                                                               | UseSecretBackend use the Agent secret backend feature for retreiving all credentials needed by the different components: Agent, Cluster, Cluster-Checks. If `useSecretBackend: true`, other credential parameters will be ignored. default value is false.                                                                                                                                                                                                                                                                                                                                                                                             |

> | `features.orchestrator_explorer.enabled`                                                                     | Enable this to activate live kubernetes monitoring on the DCA See: https://docs.datadoghq.com/infrastructure/livecontainers/#kubernetes-resources-1. Note: this features the process-agent running. Live containers are enough to be activated See: https://docs.datadoghq.com/infrastructure/process/?tab=linuxwindows. Will add the process agent and auto activate live containers if not set otherwise (disabled)                                                                                                                                                                                                                                  |

Reviewer: ```suggestion
| `features.orchestrator_explorer.enabled`                                                                     | Enable this to activate live Kubernetes monitoring on the DCA, see: https://docs.datadoghq.com/infrastructure/livecontainers/#kubernetes-resources-1. Note: This features the process Agent running. Live containers are enough to be activated, see: https://docs.datadoghq.com/infrastructure/process/?tab=linuxwindows. Adds the process Agent and auto activates live containers if not set otherwise (disabled)                                                                                                                                                                                                                                  |
```

> | `features.orchestrator_explorer.container_scrubbing.enabled`                                                 | Deactivate this to stop the scrubbing of sensitive container data (passwords, tokens ect.)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |

Reviewer: ```suggestion
| `features.orchestrator_explorer.container_scrubbing.enabled`                                                 | Deactivate this to stop the scrubbing of sensitive container data (passwords, tokens, etc.)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
```

| `features.orchestrator_explorer.extra_tags`                                                                  | Additional tags for the collected data                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |

> | `features.orchestrator_explorer.orchestrator_dd_url`                                                         | Set this for the datadog endpoint for the orchestrator explorer                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |

Reviewer: ```suggestion
| `features.orchestrator_explorer.orchestrator_dd_url`                                                         | Set this for the Datadog endpoint for the orchestrator explorer                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```
